frontend/docsettings.lua

```diff
@@ -116,6 +130,13 @@ function DocSettings:getDocSidecarFile(doc_path, no_legacy)
             return sidecar_file
         end
     end
+
+    if lfs.attributes(DOCSETTINGS_HASH_DIR, "mode") == "directory" then
+        local _, hash_sidecar_file = self:getSidecarHashDirAndFilepath(doc_path)
```

Comment:
Can be similar to the above: `sidecar_file = self:getSidecarFile(doc_path, "hash")`